apps/collabora.sh

```diff
@@ -23,6 +19,23 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+# Check if onlyoffice ist already installed
+print_text_in_color "$ICyan" "Checking if Onlyoffice or Collabora is already installed..."
+if does_this_docker_exist 'onlyoffice/documentserver' || does_this_docker_exist 'collabora/code'
+then
+    msg_box "It seems like 'Onlyoffice' or 'Collabora' is already installed.\nIf you continue, Onlyoffice will be deleted and Collabora (re-)installed."
+    if [[ "no" == $(ask_yes_or_no "Do you really want to continue?") ]]
+    then
+        exit
+    fi
+fi
+
+print_text_in_color "$ICyan" "(Re-)Installing Collabora..."
+
+NC_UPDATE=1 && COLLABORA_INSTALL=1 
```

Comment:
NC_UPDATE=1 && COLLABORA_INSTALL=1 . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)

It's stuff like this that makes me a bit worried. How can you miss the curl command? Aren't you testing your changes?

Reply:
I habe reverted that because i tested it and it didn't work ;)

Reply:
Haha ok! :D